plone/app/dexterity/behaviors/nextprevious.py

```diff
@@ -96,6 +98,8 @@ def getPreviousItem(self, obj):
             return None
         order_reversed = list(reversed(self.order))
         pos = order_reversed.index(obj.getId())
+        if not pos:
```

Comment:
index will return a number or raise a ValueError if the item is not in the list.

I am wondering if you really want to have this code:
```
if pos == 0: 
    return    
```

Reply:
pos is None. No idea why actually. "pos == 0" won't solve the problem. :(

Reply:
@ale-rt sorry, you are right about this one. I just added it kind of "just in case". We can remove it here. I discussed this with @sneridagh and we suspect that this happens only if you pass a brain to the function. The brain does not have a context and then pos is None. Unfortunately we can not check this in the plone.restapi serializer, so we have to fix this problem here. Though, maybe there is a better way to fix that than my first attempt.

Reply:
This is super odd. I have an aq wrapped object here that just does not return an object position. No idea why...